README.md

```diff
@@ -44,8 +44,8 @@ Project status
 
 The project is test with:
 * MySQL 5.5, 5.6 and 5.7
-* Python 3.3, 3.4, 3.5 and 3.6 (3.2 is not supported)
-* PyPy (really faster than the standard Python interpreter)
+* Python 3.7, 3.11 (3.7 is unsupported)
```

Comment:
The 3.7 is still supported ;)

Reply:
Yes, indeed! I removed the unsupported phrase.